include/SDL3/SDL_gamepad.h

```diff
@@ -311,25 +311,16 @@ extern DECLSPEC int SDLCALL SDL_AddGamepadMappingsFromFile(const char *file);
  */
 extern DECLSPEC int SDLCALL SDL_ReloadGamepadMappings(void);
 
-/**
- * Get the number of mappings installed.
- *
- * \returns the number of mappings.
- *
- * \since This function is available since SDL 3.0.0.
- */
-extern DECLSPEC int SDLCALL SDL_GetNumGamepadMappings(void);
-
 /**
  * Get the mapping at a particular index.
  *
- * \param mapping_index mapping index
- * \returns the mapping string. Must be freed with SDL_free(). Returns NULL if
- *          the index is out of range.
+ * \param count On return, set to the number of mappings returned. Can be NULL.
+ * \returns an array of the mapping strings, NULL-terminated. Must be freed with SDL_free().
```

Comment:
It might be worth calling out that the individual strings don't need to be freed. That was the first question I had when I looked at this function.